iconic/UserProfile/views.py

```diff
@@ -17,3 +17,42 @@ def get(self, request):
         current_wallet = Wallet.objects.get(user_id=request.user.id)
         data = CustomWalletSerializer(current_wallet).data
         return Response(data)
+
+
+# a view for changing and adding data to a resume
+class UserResumeEditView(APIView):
+    serializers_class = serializers.CustomResumeSerializer
+    permissions_classes = (permissions.IsAuthenticated)
+
+    #  for adding
+    def post(self, request):
+        data = dict(request.data)
+        data["user_id"] = request.user.id
+        serializer = CustomResumeSerializer(data)
```

Comment:
In this line you need to use kword argument like "data=data" instead of just "data"